src/script/standard.cpp

```diff
@@ -370,3 +373,99 @@ CScript GetScriptForMultisig(int nRequired, const std::vector<CPubKey>& keys)
 bool IsValidDestination(const CTxDestination& dest) {
     return dest.index() != 0;
 }
+
+/*static*/ TaprootBuilder::NodeInfo TaprootBuilder::Combine(NodeInfo&& a, NodeInfo&& b)
```

Comment:
Are these `static` keywords here and one more below intentionally commented out?

Reply:
Yes, the `static` needs to be mentioned at declaration time (in the class in the .h) file, you can't repeat it at definition time. But putting it here in comments reminds the reader that it's actually a static member function.

Reply:
I see, thanks! I had never seen this before.

Reply:
C++'s dual use of the keyword `static` is kind of confusing, but not @sipa's fault of course. I think documenting that it's a static member function is useful here.